cernopendata/modules/glossary/fetchers.py

```diff
@@ -0,0 +1,17 @@
+"""PID Fetchers."""
```

Comment:
I agree for the ```cernopendata/modules/records/glossary/```
IMHO as we follow the same style in CAP I believe that the best is not to split everything in independent packages and create a ```modules``` package since this way is more structured and everything comes directly with the main project.  What do you think of that?